CHANGELOG.md

```diff
@@ -9,11 +9,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [0.3.1] - 2022-02-07
+## [0.4.0] - 2022-02-08
```

Comment:
Changelog looks weird but this is basically because I did a stupid when merging my PR which got the changelog messed up. Should be correct now.